api/v2alpha1/google/rpc/status.proto

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
```

Comment:
This file isn't available via protoc's standard include path. Having a local copy (forked from https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto) seems to be the simplest approach.
But let me know if theres's a better solution.

Reply:
You could potentially use a Makefile to copy during compilation with protoc. That way, you can specify a version. Copying locally seems ok too. @Bobgy may have some ideas on this as well.

Reply:
Missed this comment before merging. Yes, I can do it this way as well.